packages/auth/src/config/app.ts

```diff
@@ -43,6 +43,7 @@ export const Config = {
   authPort: envInt('AUTH_PORT', 3006),
   interactionPort: envInt('INTERACTION_PORT', 3009),
   introspectionPort: envInt('INTROSPECTION_PORT', 3007),
+  serviceAPIPort: envInt('SERVICE_API_PORT', 3011),
```

Comment:
what do you think about calling this Tenant service instead? 

Are you approaching this from this service being used for something else, for example?

Reply:
> Are you approaching this from this service being used for something else, for example?

Yes, that was the idea. Figured it can serve the purpose of any service-to-service communication. No specific things in mind but if we did want to expose some other resource I dont imagine we'd want a new api just for that.